server/sensor-helm-chart/templates/sensor-deployment.yaml

```diff
@@ -165,6 +165,11 @@ spec:
               secretKeyRef:
                 name: {{ .Values.secretName | quote }}
                 key: CLICKUP_TEAM_ID
+          - name: MAX_STILLNESS_ALERTS
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.secretName | quote }}
+                key: MAX_STILLNESS_ALERTS
```

Comment:
Yay! Remembered to do this!

When you deploy it on dev, we'll also need to remember to pull your branch on Sensors Admin. That was the other thing that we forgot last time.